pipeline/integration/speciesDataProcessing.R

```diff
@@ -113,7 +114,9 @@ if (uploadToWallace == TRUE) {
 
 # Here we see which species have sufficient presence/count data to actually run an individual species model
 redListSpecies <- filterByRedList(redList$GBIFName, processedPresenceData, 5)
-saveRDS(redListSpecies, paste0(folderName, "/redList.RDS"))
+redList$valid <- ifelse(redList$GBIFName %in% redListSpecies$validSpecies, TRUE, FALSE)
```

Comment:
why not just:
redList$valid <- redList$GBIFName %in% redListSpecies$validSpecies